creator-node/src/components/healthCheck/healthCheckController.js

```diff
@@ -79,12 +91,27 @@ const healthCheckVerboseController = async (req) => {
   })
 }
 
+/**
+ * Controller for `health_check/fileupload` route
+ * Calls `healthCheckFileUploadService`.
```

Comment:
where does this call `healthCheckFileUploadService`?

also would be good to document a bit more what this does, ie confirms a large file can be uploaded and prunes all disk artifacts after etc

Reply:
Sorry, it used to, removed that part. Updated with a comment line explaining the file upload and prune.